pinot-integration-tests/pom.xml

```diff
@@ -237,8 +241,21 @@
           <groupId>commons-lang</groupId>
           <artifactId>commons-lang</artifactId>
         </exclusion>
+        <exclusion>
+          <groupId>org.apache.zookeeper</groupId>
+          <artifactId>zookeeper</artifactId>
+        </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>com.sun.jersey</groupId>
+      <artifactId>jersey-core</artifactId>
+      <version>1.9</version>
```

Comment:
why did we add this dependency with a version? Should versions be enforced at the top level?

Reply:
It seems `jersey-core` has been brought from hadoop-minicluster jar. We don't need to specify it here any more. I've removed it in the latest push. It's been tested.